java/src/jmri/util/exceptionhandler/UncaughtExceptionHandler.java

```diff
@@ -25,7 +27,15 @@ public void uncaughtException(Thread t, Throwable e) {
             return;
         }
 
-        log.error("Unhandled Exception: " + e, e);
+        log.error("Uncaught Exception: {}", generateStackTrace(e));
```

Comment:
@rhwood The two formats are similar, but not exactly the same.  
error(msg, e) outputs the message separate lines after the error log entry:
````
ERROR - form with comma: [testThread] jmri.util.exceptionhandler.UncaughtExceptionHandler.?()
java.lang.NullPointerException
	at jmri.util.exceptionhandler.UncaughtExceptionHandlerTest$1.deref(UncaughtExceptionHandlerTest.java:27)
	at jmri.util.exceptionhandler.UncaughtExceptionHandlerTest$1.run(UncaughtExceptionHandlerTest.java:23)
````
The form with the {} in the message string inserts the exception and stack trace in a single log entry that can be processed as a single item later by e.g. the JUnit test compare-for-correct-message method.
````
ERROR - format with braces: java.lang.NullPointerException
	at jmri.util.exceptionhandler.UncaughtExceptionHandlerTest$1.deref(UncaughtExceptionHandlerTest.java:27)
	at jmri.util.exceptionhandler.UncaughtExceptionHandlerTest$1.run(UncaughtExceptionHandlerTest.java:23)
 [testThread] jmri.util.exceptionhandler.UncaughtExceptionHandler.?()
````